models.py

```diff
@@ -12,3 +16,35 @@ def mostrecent(x):
     preds = X.apply(mostrecent, axis=1)
 
     return preds
+
+def arima(X):
+    """Predict the most recent value using ARIMA(2,1,0) model.
+
+    If any issues fitting time series model, use status_quo."""
+    
+    sq_preds = status_quo_model(X)
+    X["id"] = X.index
+    # get it to long form
+    longX = pd.melt(X, id_vars=['id'], value_vars=list(X.columns.values)[:-1])
+    # change the default column name to a proper one
+    longX['year'] = longX['variable']
+    longX.drop(columns = 'variable', inplace=True)
+    # sort by id and year so that we can fit individual time series models for each indicator
+    longX = longX.sort_values(['id', 'year'])
+    # group
+    grouped = longX.groupby('id')['value']
+    # for each group (a country-indicator combination)
```

Comment:
I'm a bit confused why this preprocessing is necessary. If the end goal is to separate each country-indicator combination into a separate group, why not just a select a row from X? Each row from X is already a sequence of values for a country-indicator combination.